src/main/java/virtual_pet/VirtualPetApplication.java

```diff
@@ -1,9 +1,106 @@
 package virtual_pet;
 
+import java.util.Scanner;
+
 public class VirtualPetApplication {
 
+
     public static void main(String[] args) {
-        //Interact with a VirtualPet object in this method
+        //Interact with a virtual_pet.VirtualPet object in this method
+        int select;
+        Scanner input = new Scanner(System.in);
+        System.out.println("What is your cats name?");
+        String catName = input.nextLine();
+
+        VirtualPet cat = new VirtualPet(catName);
```

Comment:
Instantiation of your object cat, should have been passed all values associated with the cat that you had initialized within the constructor in VirtualPet.  Please see comment in VirutalPet about the VirtualPet constructor.